packages/@aws-cdk/aws-appmesh/lib/index.ts

```diff
@@ -15,6 +15,7 @@ export * from './virtual-gateway';
 export * from './virtual-gateway-listener';
 export * from './gateway-route';
 export * from './gateway-route-spec';
-export * from './client-policy';
 export * from './health-checks';
 export * from './tls-listener';
+export * from './tls-validation';
+export { TlsClientPolicy } from './client-policy';
```

Comment:
@skinny85 - I am actually curious if this is better statement than saying `export * from` when the module is only exporting one item.

Reply:
No. Because then you have to consult a separate file to know whether a given exported member is part of the API of the module, or not.

This way, all that information is kept local to the file the class/interface is defined in.

Reply:
Actually, I believe the build might be failing because of this (it might be a JSII quirk that requires `export *`).